lms/djangoapps/grades/management/commands/backfill_grades.py

```diff
@@ -0,0 +1,141 @@
+"""
+Command to compute all grades for specified courses.
+"""
+
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import logging
+
+from django.core.management.base import BaseCommand, CommandError
+import six
+
+from openedx.core.lib.command_utils import (
+    get_mutually_exclusive_required_option,
+    validate_dependent_option,
+    parse_course_keys,
+)
+from student.models import CourseEnrollment
+from xmodule.modulestore.django import modulestore
+
+from ... import tasks
+
+
+log = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
```

Comment:
The command needs to allow the values for `batch_size` and `course_ids` to be overridden by django admin settings.  Will that be done in a follow-up PR?